src/metadataGeneration/parameterGenerator.ts

```diff
@@ -31,6 +31,8 @@ export class ParameterGenerator {
         return this.getQueryParameter(this.parameter);
       case 'Path':
         return this.getPathParameter(this.parameter);
+      case 'Ignore':
+        return null;
```

Comment:
I don't think we should change the return type of `Generate` to `Tsoa.Parameter | null`. I think that if you want to ignore a property we have one of two options we can go with:

1) if the goal of the decorator to ignore the validation, then we should rename the decorator to be `IgnoreValidation` and then we should enforce that the `type.dataType` is equal to `any` since that's what the user is basically saying to us (that they don't want any type correctness).
2) if the goal of the decorator is to just not have to send the parameter, then it really begs the question of why does the route even have the query parameter? Anyway, if that's the goal, then we should still return a `Tsoa.Parameter` but it would return this:
```ts
case 'Ignore':
  return {
    //...
    type: {
      dataType: 'any',
    },
    validators: {},
    required: false
}
```